Trakt/Extensions.cs

```diff
@@ -161,12 +175,12 @@ public static bool MetadataIsDifferent(this TraktMovieCollected collectedMovie,
         }
 
         /// <summary>
-        /// Gets the ISO-8620 representation of a <see cref="DateTime"/>.
+        /// Gets the ISO-8601 representation of a <see cref="DateTime"/>.
         /// </summary>
-        /// <param name="dt">The <see cref="DateTime"/>.</param>
+        /// <param name="dateTime">The <see cref="DateTime"/>.</param>
         /// <returns>string.</returns>
-        public static string ToISO8601(this DateTime dt)
-            => dt.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ", CultureInfo.InvariantCulture);
+        public static string ToISO8601(this DateTime dateTime)
+            => dateTime.ToUniversalTime().ToString("yyyy-MM-dd'T'HH:mm:ss.fffffff'Z'", CultureInfo.InvariantCulture);
```

Comment:
ToString("o") is equivalent btw

Reply:
Does `ToString("o")` really do `HH:mm:sssZ` and not `HH:mm:ssZ` ?

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#Roundtrip